examples/evm/use-erc20/src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import web3 from "web3";
+import { Contract, getDefaultProvider, Wallet } from "ethers";
+
+export const assetIdToERC20ContractAddress = (
+  assetId: string | number
+): string => {
+  const asset_id_hex = (+assetId).toString(16).padStart(8, "0");
+  return web3.utils.toChecksumAddress(
```

Comment:
You can use `ethers.utils.getAddress` here rather than adding a new library

```suggestion
  const asset_id_hex = (+assetId).toString(16).padStart(8, "0").toUpperCase();
  return ethers.utils.getAddress(
```

Reply:
I tried using this and got error.. so ended up using web3.

```
import web3 from "web3";
import { stringToHex } from "@polkadot/util";
import {getAddress} from "ethers/lib/utils";

export const assetIdToERC20ContractAddress = (assetId: string | number): string => {
	const asset_id_hex = (+assetId).toString(16).padStart(8, "0");
	console.log('asset_id_hex:',asset_id_hex);
	return web3.utils.toChecksumAddress(`0xCCCCCCCC${asset_id_hex}000000000000000000000000`);
};


export const assetIdToERC20ContractAddress1 = (assetId: string | number): string => {
	const asset_id_hex = (+assetId).toString(16).padStart(8, "0");
	console.log('asset_id_hex:',asset_id_hex);
	return getAddress(`0xCCCCCCCC${asset_id_hex}000000000000000000000000`);
};

console.log('1st assetIdToERC20ContractAddress::',assetIdToERC20ContractAddress(12));

console.log('2nd assetIdToERC20ContractAddress::',assetIdToERC20ContractAddress1(12));

```

Reply:
<img width="1497" alt="Screenshot 2023-07-07 at 8 21 43 AM" src="https://github.com/futureversecom/trn-examples/assets/29415595/3e51f224-6a7c-4024-99de-1e23192d815c">

Reply:
It should work with 
```
getAddress(`0xCCCCCCCC${asset_id_hex.toUpperCase()}000000000000000000000000`)
```

Reply:
works!! thank you